### ISSUE TITLE
Incorrect attribute aria-labeledby in dropdown

### ISSUE THREAD

**Issue by mackensen** [Formstone/Formstone#239]
In src/js/dropdown.js the attribute `aria-labeledby` is set; it should be `aria-labelledby`. This causes a WCAG 4.1.2 violation in any downstream use of the dropdown.
